[2025-10-10T15:39:21.975+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-10T15:39:21.997+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_Lab1.load_model_task manual__2025-10-10T15:26:15.256544+00:00 [queued]>
[2025-10-10T15:39:22.006+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_Lab1.load_model_task manual__2025-10-10T15:26:15.256544+00:00 [queued]>
[2025-10-10T15:39:22.006+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-10T15:39:22.021+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_model_task> on 2025-10-10 15:26:15.256544+00:00
[2025-10-10T15:39:22.033+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1031) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-10T15:39:22.035+0000] {standard_task_runner.py:63} INFO - Started process 1061 to run task
[2025-10-10T15:39:22.035+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Airflow_Lab1', 'load_model_task', 'manual__2025-10-10T15:26:15.256544+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/***1.py', '--cfg-path', '/tmp/tmp8wqludmq']
[2025-10-10T15:39:22.038+0000] {standard_task_runner.py:91} INFO - Job 14: Subtask load_model_task
[2025-10-10T15:39:22.129+0000] {task_command.py:426} INFO - Running <TaskInstance: Airflow_Lab1.load_model_task manual__2025-10-10T15:26:15.256544+00:00 [running]> on host 9ecfad8aaa0c
[2025-10-10T15:39:22.229+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='your_name' AIRFLOW_CTX_DAG_ID='Airflow_Lab1' AIRFLOW_CTX_TASK_ID='load_model_task' AIRFLOW_CTX_EXECUTION_DATE='2025-10-10T15:26:15.256544+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-10T15:26:15.256544+00:00'
[2025-10-10T15:39:22.232+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-10T15:39:22.252+0000] {logging_mixin.py:188} INFO - Optimal no. of clusters: 8
[2025-10-10T15:39:22.265+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/sklearn/utils/validation.py:2742: UserWarning: X has feature names, but KMeans was fitted without feature names
  warnings.warn(

[2025-10-10T15:39:22.305+0000] {python.py:237} INFO - Done. Returned value was: 5
[2025-10-10T15:39:22.312+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-10T15:39:22.374+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Airflow_Lab1, task_id=load_model_task, run_id=manual__2025-10-10T15:26:15.256544+00:00, execution_date=20251010T152615, start_date=20251010T153921, end_date=20251010T153922
[2025-10-10T15:39:22.412+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-10T15:39:22.432+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-10T15:39:22.435+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
